docs/usage/export.md

```diff
@@ -1,5 +1,57 @@
-Exporting data
+Exporting Data
 ==============
 
 ## Introduction
-Feature coming soon.
\ No newline at end of file
+
+The Bulk Export feature allows you to export data from a Sleeper table to an S3 bucket in a number of Parquet files.
+
+## Prerequisites
+Before performing a Bulk Export, ensure that the `BulkExportStack` has been deployed. This stack is optional and is not included in a vanilla installation.
+
+## How to Perform a Bulk Export
+
+### Step 1: Submit a Bulk Export Query
+
+To initiate a Bulk Export, submit a **Bulk Export Query** to the SQS queue specified in the `BULK_EXPORT_QUEUE_URL` property. The query should specify the Sleeper table name or table id to be exported. If wanted an `exportId` can also be provided, if one isn't then one is generated.
```

Comment:
The property name is `sleeper.bulk.export.queue.url`, not `BULK_EXPORT_QUEUE_URL`.

I read this as specified by the user, but it's actually set by the CDK. I'm not sure if it's just me, but maybe just removing the word "specified" would help.